File: tmp/copy_image.js
const fs = require('fs');
const path = require('path');

const source = 'C:\\Users\\grace\\.gemini\\antigravity\\brain\\976224f1-f9df-41b8-be50-ff35a8d02d76\\media__1773059709426.jpg';
const dest = 'c:\\Users\\grace\\grace-isitua\\public\\grace-portrait.jpg';

try {
  fs.copyFileSync(source, dest);
  console.log('Successfully copied ' + source + ' to ' + dest);
} catch (err) {
  console.error('Error copying file:', err);
  process.exit(1);
}

File: copy_image.py
import shutil
import os

source = r"C:\Users\grace\.gemini\antigravity\brain\976224f1-f9df-41b8-be50-ff35a8d02d76\media__1773059709426.jpg"
dest = r"c:\Users\grace\grace-isitua\public\grace-portrait.jpg"

try:
    if os.path.exists(source):
        print(f"File exists at source. Size: {os.path.getsize(source)}")
        shutil.copy2(source, dest)
        print(f"Successfully copied to {dest}")
    else:
        print(f"Source file not found: {source}")
except Exception as e:
    print(f"Error copying: {e}")
